src/tests/topp/SimpleSearchEngine_1_out.idXML

```diff
@@ -18,14 +18,14 @@
 			<UserParam type="stringList" name="spectra_data" value="[file://SimpleSearchEngine_1.mzML]"/>
 		</ProteinIdentification>
 		<PeptideIdentification score_type="hyperscore" higher_score_better="true" significance_threshold="0.0" MZ="520.262817382812045" RT="2655.095703125" >
-			<PeptideHit score="20.34238715336474" sequence="DFASSGGYVLHLHR" charge="3" aa_before="[" aa_after="E" start="0" end="13" protein_refs="PH_0" >
+			<PeptideHit score="12.094764845630944" sequence="DFASSGGYVLHLHR" charge="3" aa_before="[" aa_after="E" start="0" end="13" protein_refs="PH_0" >
```

Comment:
why are these results different @timosachsenberg is that expected ?

Reply:
@enetz this is indeed a bit strange that the hyper score decreases that much. Can you check

Reply:
@enetz can you check. means that significantly less b/y ions have been matched

Reply:
@timosachsenberg I just ran this test with `start_intensity_check = 1`, which is effectively the old default averagine model. The score is then `19.24...`. This smaller left over difference should be due the precursor charge / mass filter.
So the score dropped by loosening the decreasing intensity requirement. This probably lead to accepting some lower intensity and lower mass peaks as new monoisotopic peaks for previously matched masses.
Since the SimpleSearchEngine does not write out peak annotations or numbers of matching peaks, it is difficult to confirm what is happening there and which score is more correct. Do you have a ground truth for these spectra? We could adjust the parameter to the old behaviour, if you think it is more correct.

Reply:
Ok, I see. Maybe we should give this a test on a larger dataset just to confirm that the new behavior does not lead to a large drop in ids at the same e.g. 1% PSM FDR.

Reply:
Eugen did some additional tests and the ID rate did not chagne on the iPRG2015.
So I guess it is fine to merge.